assignment.js

```diff
@@ -66,6 +74,9 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+let values = [3, 48, 29, 12, 18]
+console.log(Math.max(...values))
```

Comment:
@rvaswani85 nice use of the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) (`...`), this is an advanced technique, just make sure you understand how it works if you choose to continue using it in the future